api/app/src/main/kotlin/packit/controllers/UserController.kt

```diff
@@ -11,23 +11,18 @@ import org.springframework.web.bind.annotation.RequestMapping
 import packit.AppConfig
 import packit.exceptions.PackitException
 import packit.model.CreateBasicUser
-import packit.security.Role
 import packit.service.UserService
 
 @Controller
 @RequestMapping("/user")
 class UserController(private val config: AppConfig, private val userService: UserService)
 {
     @PostMapping("/basic")
+    @PreAuthorize("hasAuthority('ADMIN')")
```

Comment:
Starting to feel like roles like "ADMIN" should be coming from a type rather than have hardcoded strings in multiple places.

Reply:
will have to be changes in next pr so have left